out/build/x64-Debug/thirdparty/glfw3/src/glfw3.pc

```diff
@@ -0,0 +1,13 @@
+prefix=C:/University/BTech TERM 3/COMP 8551/EngineProjectRedo/GameEngine/out/install/x64-Debug
```

Comment:
What's the purpose of these files? where these missing from the previous GLFW3 install?